src/app/search/search-filters/simple.tsx

```diff
@@ -6,6 +6,12 @@ import { FilterDefinition } from '../filter-types';
 
 // simple checks against check an attribute found on DimItem
 const simpleFilters: FilterDefinition[] = [
+  {
+    keywords: 'adept',
+    description: tl('Filter.Adept'),
+    destinyVersion: 2,
```

Comment:
Is this is meant to work for D1 too, `destinyVersion` should be omitted.

Reply:
Removed `destinyVersion` from the `adept`, `harrowed`, and `timelost` filters implemented in this pull request.